### PR TITLE
update vendor for mips64 and mips64el

### PR DESCRIPTION
**Pull request by wanghuaiqing2010**
Signed-off-by: wanghuaiqing <wanghuaiqing@loongson.cn>

### CODE REVIEW

**Comment by wanghuaiqing2010** [comment]
@ tonistiigi PTAL

File: go.mod [hunk 1]
```diff
@@ -3,9 +3,9 @@ module github.com/tonistiigi/binfmt
...
 require (
-	github.com/containerd/containerd v1.4.3
-	github.com/moby/buildkit v0.8.0
+	github.com/containerd/containerd v1.4.1-0.20201215193253-e922d5553d12
+	github.com/moby/buildkit v0.8.2-0.20210202025417-7cd6a5feaf83
 	github.com/pkg/errors v0.9.1
 )
 
-replace github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200227233006-38f52c9fec82
+replace github.com/containerd/stargz-snapshotter/estargz v0.0.0-00010101000000-000000000000 => github.com/containerd/stargz-snapshotter/estargz v0.0.0-20210202123615-f0962e4437ca
```
Comment (tonistiigi) [inline go.mod:11]: Hmm, `estargz` does not seem to be used by this repo. So why is this line needed? Also, no need to update containerd version in this file if we don't actually need any functional updates (gomod recommendation is to define minimal required version).

**Comment by wanghuaiqing2010** [comment]
@tonistiigi 
If I do not add estargz, the following error will occur.
Please check it.


```
[root@whq binfmt]# go run cmd/binfmt/config.go cmd/binfmt/main.go
go: github.com/moby/buildkit@v0.8.2-0.20210202025417-7cd6a5feaf83 requires
	github.com/containerd/stargz-snapshotter@v0.3.0 requires
	github.com/containerd/stargz-snapshotter/estargz@v0.0.0-00010101000000-000000000000: invalid version: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/gopath/pkg/mod/cache/vcs/0c47353239954145e83a4629fd4c83b84eb1e06381df0c483f130ea80d8e0232: exit status 128:
	error: RPC failed; result=18, HTTP code = 200
	fatal: The remote end hung up unexpectedly
	fatal: 过早的文件结束符（EOF）
	fatal: index-pack failed
```

**Comment by wanghuaiqing2010** [comment]
@tonistiigi I have added mips64le to image tonistiigi/binfmt, please review it.
https://github.com/tonistiigi/binfmt/pull/21

**Comment by tonistiigi** [comment]
@ktock fyi ^

**Comment by ktock** [comment]
Currently, we have two go.mod in one repo so `replace` directive is needed if the project has a dependency to `github.com/containerd/stargz-snapshotter` on go.mod level, even if it doesn't actually use codes from that repo.

We start to track this issue on https://github.com/containerd/stargz-snapshotter/issues/259.
At this moment, please add the `replace` directive and we'll remove this once that issue is solved.

**Review by tonistiigi — APPROVED** [review]
